imessage/bluebubbles/api.go

```diff
@@ -122,7 +122,13 @@ func (bb *blueBubbles) PollForWebsocketMessages() {
 			if websocket.IsUnexpectedCloseError(err, websocket.CloseGoingAway, websocket.CloseAbnormalClosure) {
 				bb.log.Error().Err(err).Msg("Error reading message from BlueBubbles websocket")
 			}
+			bb.log.Info().Msg("Attempting to reconnect to BlueBubbles websocket...")
+            err = bb.retryConnect()
```

Comment:
I'm not seeing the `retryConnect` function anywhere, do you have to make one?
 - I think you should probably do everything in lines 87-97 in `retryConnect`, the initial handshake is important
 - also would be a good idea to bake in a 5-10 sec sleep and Max retries. if the BB URL changes, which it does often, you don't want this looping aggressively forever

Reply:
@cnuss we are actually talking about this in the mautrix-imessage channel already. ChatGPT just hallucinated on @rollingonchrome